src/Microsoft.Performance.SDK.Runtime/DTO/TableConfigurationsSerializer.cs

```diff
@@ -146,27 +154,129 @@ public IEnumerable<Processing.TableConfigurations> DeserializeTableConfiguration
         }
 
         /// <summary>
-        ///     Serializes a table configuration to a stream.
+        ///     Serializes one or more <see cref="Processing.TableConfiguration"/> instances to a stream.
+        /// </summary>
+        /// <param name="stream">
+        ///     Target stream.
+        /// </param>
+        /// <param name="tableConfigurations">
+        ///     The <see cref="Processing.TableConfiguration"/> instances to serialize.
+        /// </param>
+        /// <param name="tableId">
+        ///     Table identifier.
+        /// </param>
+        /// <remarks>
+        ///     This will serialize a full <see cref="Processing.TableConfigurations"/> that has the given
+        ///     <paramref name="tableId" /> as its <see cref="Processing.TableConfigurations.TableId"/>
+        ///     and the given <paramref name="tableConfigurations"/> as its <see cref="Processing.TableConfigurations.Configurations"/>.
+        ///     The serialized <see cref="Processing.TableConfigurations"/> will have a null
+        ///     <see cref="Processing.TableConfigurations.DefaultConfigurationName"/>.
+        /// </remarks>
+        public static void SerializeTableConfigurations(
+            Stream stream,
+            Processing.TableConfiguration[] tableConfigurations,
```

Comment:
>Processing.TableConfiguration[] [](http://example.com/codeflow?start=12&length=31)

Any reason not to make this IEnumerable<> instead of an array?

Reply:
Nope, updated.